roles/v2v.ci/tasks/rhv_install_nmon.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+- name: Add {{ rhv_item }}
+  add_host:
+    name: "{{ rhv_item }}"
+    ansible_user: root
+    ansible_ssh_pass: "{{ hostvars[rhv_item].root_password }}"
+    ansible_pipelining: true
+
+# Install Nmon
+- name: Gathering facts
+  delegate_to: "{{ rhv_item }}"
+  setup:
+
+- name: Copy {{ nmon_file_rhel7 }} to {{ rhv_item }}
```

Comment:
this copy task and the next one are the same except the nmon_file_rhelX
maybe we can set fact before like:
set_fact:
  nmon_file: "{{ nmon_file_rhel7 if ansible_distribution_major_version == '7' else  nmon_file_rhel8 }}"
and use one copy task with the new fact nmon_file

Reply:
It is a good idea. Thanks
But still the last task - 'Ensure ncurses packages are installed' is related only to rhel8, so we need the when condition for it

Reply:
yep, you can give up on the "block" and add the "when" condition only to this task.
the main principle that you need to think of, is the maintenance of the code.
if you need to change something in one task now you need doing it in one place and not in a few(which can save problems in the future)